tests/acceptance/features/bootstrap/SpacesContext.php

```diff
@@ -759,11 +773,7 @@ public function rememberTheAvailableSpaces(?ResponseInterface $response = null):
 	 * @return void
 	 * @throws GuzzleException
 	 */
-	public function theUserListsTheContentOfAPersonalSpaceRootUsingTheWebDAvApi(
-		string $user,
-		string $spaceName,
-		string $foldersPath = ''
-	): void {
+	public function listContentOfSpace(string $user, string $spaceName, string $foldersPath = ''): void {
 		$this->setSpaceIDByName($user, $spaceName);
 		$this->featureContext->setResponse(
 			WebDavHelper::propfind(
```

Comment:
helper methods must not use `setResponse`. Please, use `setResponse` only within the step def methods
```suggestion
		return WebDavHelper::propfind(
```